activemq/CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # CHANGELOG - activemq
 
-0.1.0/ Unreleased
+1.0.0/ Unreleased
```

Comment:
Do you think we might want to add a note or footer in the individual changelogs pointing to the `dd-agent` changelog?